src/gpu/intel/jit/conv/problem.cpp

```diff
@@ -45,6 +45,62 @@ const std::vector<pvar_t> &conv_dims() {
     return _conv_dims;
 }
 
+pvar_t prb_stride(pvar_t dim, tensor_kind_t tensor_kind) {
+    auto ret = pvars::src_mb_stride;
+    if (tensor_kind == tensor_kind_t::src) {
+
+        if (dim == pvars::mb)
+            ret = pvars::src_mb_stride;
```

Comment:
For simplicity we can handle it generically `return pvar_t("src_" + dim.str() + "_stride");`